crypto/algorithms/src/main/java/org/hyperledger/besu/crypto/CodeDelegationSignature.java

```diff
@@ -45,6 +45,11 @@ public static CodeDelegationSignature create(
       final BigInteger r, final BigInteger s, final byte yParity) {
     checkNotNull(r);
     checkNotNull(s);
+    final byte recId = yParity;
+    if (recId != 0 && recId != 1) {
```

Comment:
I think that is not correct. The 7702 spec says that the transaction is valid as long as recId is less than 2**8: https://eips.ethereum.org/EIPS/eip-7702#set-code-transaction

So we should process the transaction, but the authorization itself would be invalid. With the current implementation we would reject transactions that other clients might accept and consider blocks invalid that other clients might consider valid.

I think the right place to put this check is in`AbstractEngineNewPayload:precomputeAuthorities`. When recId isn't 0 or 1, we should skip to precompute the authority.